### ISSUE TITLE
qtile-cmd : unkown command

### ISSUE THREAD

**Issue by Rockwell1799** [qtile/qtile#2566]
# Issue description
I came back to qtile after few monts of Awesomewm and before I was able to run this commad in my terminal:
qtile-cmd
now, It says that this command is unkown!
I tried also:
qtile run-cmd and it gives this
```
Traceback (most recent call last):
  File "/usr/bin/qtile", line 33, in <module>
    sys.exit(load_entry_point('qtile==0.17.0', 'console_scripts', 'qtile')())
  File "/usr/lib/python3.9/site-packages/libqtile/scripts/main.py", line 59, in main
    options.func(options)
  File "/usr/lib/python3.9/site-packages/libqtile/scripts/run_cmd.py", line 41, in run_cmd
    proc = subprocess.Popen(opts.cmd)
  File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.9/subprocess.py", line 1691, in _execute_child
    executable = args[0]
IndexError: list index out of range
```
os:Archlinux

How can I fix this issue, please?

# Qtile version
Qtile 0.17.01 (official)

# Stack traces

# Configuration

**Comment by hackedWifi** [comment]
Here is some information on these commands.
http://docs.qtile.org/en/latest/manual/commands/shell/qtile-run.html

**Comment by elParaguayo** [comment]
I can confirm I get the same error where it's run with no parameters. This shouldn't happen and, instead, it should provide more info on how to use the command.

Will mark as a bug for now.

**Comment by nazar-pc** [comment]
This command no longer exist, you need to use `qtile cmd-obj`, see changelog for 0.17.0: https://github.com/qtile/qtile/blob/master/CHANGELOG

**Comment by elParaguayo** [comment]
Indeed, the changes were as follows:
```
                qtile     -> qtile start
                qtile-cmd -> qtile cmd-obj
                qtile-run -> qtile run-cmd
                qtile-top -> qtile top
                qshell    -> qtile shell
```
However, running `qtile run-cmd` should provide some more useful output when run, rather than crashing.

**Comment by elParaguayo** [comment]
Closed by #2583
